content/news.pml

```diff
@@ -39,6 +39,63 @@
 
 <!----------------------------------------------------------->
 
+<div class="news-item">
+  28 July 2021: Proof that seL4 enforces integrity established for RISC-V
+</div>
+<div>
+ <p>
+   <img src="/Foundation/Membership/LOGOS/RISC-V.svg"
+   style="width: 20%;  padding-left:10px;  float:right"
+	alt="RISC-V logo">
+
+   The assurance story for <a href="https://sel4.systems">seL4</a>
+   on <a href="https://riscv.org">RISC-V</a> keeps building. We first
+   formally proved <em>functional
+   correctness</em>: <a href="https://microkerneldude.wordpress.com/2020/06/09/sel4-is-verified-on-risc-v/">that
+   the seL4 C code on RISC-V platforms behaves exactly as its
+   specification says</a>. We then established <em>binary
```

Comment:
Maybe re-word this to "Then, we established ..."

Reply:
same as above